### ISSUE TITLE
Editor pane morphs during PECE essay creation

### ISSUE THREAD

**Issue by akenner** [PECE-project/pece-distro#91]
On housingenergy.info, when we are working on PECE essays, and we go to add a new tile [by clicking the plus sign at one of the three top columns] all the sudden we will get some wild admin upgrade. I have the two screen shots below. SS1 = normal editor pane. SS2 = weird bonus round editor pane upgrade... but that doesn't include the needed features as the normal editor pane. Such as 'content heading' etc.

What is SSN2? We are able to revert to normal by saving, logging out and then back in. 
Is this a bug?

**SS1**
<img width="1081" alt="Screen Shot 2020-06-02 at 8 38 50 PM" src="https://user-images.githubusercontent.com/1063084/83583025-375b9f80-a511-11ea-9150-d569b85626df.png">

**SS2**
<img width="1034" alt="Screen Shot 2020-06-02 at 8 28 15 PM" src="https://user-images.githubusercontent.com/1063084/83583030-3a569000-a511-11ea-9146-b3787ed05a4d.png">

**Comment by revagomes** [comment]
@akenner,

**SS2** is the full featured configuration pane for customizing Essay pages. 
It should to be available only for users with Administrator role.

Are you having this behavior with a Researcher or Contributor user?

**Comment by akenner** [comment]
@revagomes Sorry for the delay!

So, I've listed all student users as both researcher and contributor. Myself and one other person are the only admins on the site. I guess I need to read the docs and see which role is best so I'm not assigning both at the same time? Is that why this is happening?

Is there documentation somewhere on how to use the full featured configuration pane, like a youtube video or anything?

**Comment by akenner** [comment]
@revagomes any idea how we can prevent this from happening? Do we need to edit the user type -- from both contributor and researcher to just one or the other?

**Comment by revagomes** [comment]
@akenner it is still not clear to me how it is actually happening. It shouldn't be happening that often anyways. 

Could you please send me a detailed step-by-step routine of the workflow you have done that ends up having this Edit pane morph?
This way I'll be able to replicate it here and then find ways to prevent that from happening. 

Regarding documentation on how to use the advanced configuration pane, we don't have documentation for that level of management yet.

**Comment by akenner** [comment]
@revagomes we have put a pause on doing PECE essays since the academic term ended. I'm going to follow-up with the current group of students to let them know about the problem and to see if this ever happens to them so that we can be prepared to document it more closely in the future. 

All I know right now, is that it happens when site users are working on a PECE essay and they click the plus button in one of the three columns to add more content. Beyond that information, I will need to investigate further to get more nuanced about the workflow. More soon!

**Comment by akenner** [comment]
@revagomes OK, so that was quicker than I thought.... 
1) This morning I went into [this PECE essay](https://housingenergy.info/?q=content/energy-covid-19-research-brief-07172020/essay), created by James Adams on the EVP instance. 
2) I click customize page, and successfully added an image tile to the third column.
3) Then I saved the essay.
4) then I hit customize page again.
5) I clicked the plus sign to add content to the second column.
6) then this window appeared. 
<img width="1285" alt="Screen Shot 2020-07-08 at 10 56 39 AM" src="https://user-images.githubusercontent.com/1063084/86934730-0b33cf80-c10a-11ea-882f-0b9a57f4d60c.png">

We'll continue to monitor these occurrences to see if we can determine a pattern.
